CMakeLists.txt

```
@@ -0,0 +1,16 @@
+project(asio-rawsockets-example)
+cmake_minimum_required(VERSION 3.2)
+
+add_compile_options(-std=c++11)
+
+find_package(Boost REQUIRED system)
+
+INCLUDE_DIRECTORIES(${Boost_INCLUDE_DIR})
+
+# Building server
+add_executable(example
+    ${CMAKE_CURRENT_SOURCE_DIR}/src/example.cpp
+)
+
```

Here we can use `target_include_directories()` instead of `INCLUDE_DIRECTORIES()` above:

```suggestion
target_include_directories(example PRIVATE ${Boost_INCLUDE_DIR})
```

---

I really don't know the difference tbh

---

I described that in another comment above.